### ISSUE TITLE
Contoller not reset between runs

### ISSUE THREAD

**Issue by grekiki2** [commaai/controls_challenge#6]
Referencing the line at https://github.com/commaai/controls_challenge/blob/ec681067d3e0650ce43843f9e2d30ae5b1ad07e1/eval.py#L75

If I implement a basic PID controller for example, the integral that I store will persist over runs degrading the controller quality. I think it would be better to reinitialize the controllers inside the loop at https://github.com/commaai/controls_challenge/blob/ec681067d3e0650ce43843f9e2d30ae5b1ad07e1/eval.py#L84

**Comment by nuwandavek** [comment]
Yup, this is true. Example controllers did not have any "state" so missed this.

**Comment by nuwandavek** [comment]
This should fix it: https://github.com/commaai/controls_challenge/commit/0ee669aae5cc84bf3cf9fa529fea6f4e7464e447
